setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='shipitapi',
-    version='0.1.0',
+    version='0.2.0',
```

Comment:
Even though this is non-breaking-API changes, I'd still promote a `1.0.0` as we've been using this in production for quite a while.  And now with Ship-it v2 incoming, I think it's worth an upgrade, but am fine with `0.2.0` too 😃